README.md

```diff
@@ -7,7 +7,7 @@ An open source interface for Uniswap -- a protocol for decentralized exchange of
 - Twitter: [@DarkFlorist](https://twitter.com/DarkFlorist)
 - Discord: [Dark Florists](https://discord.com/invite/aCSKcvf5VW)
 - Whitepapers:
-  - [V1](https://hackmd.io/C-DvwDSfSxuh-Gd4WKE_ig)
-  - [V2](https://uniswap.org/whitepaper.pdf)
-  - [V3](https://uniswap.org/whitepaper-v3.pdf)
+  - [V1](https://bafybeihyq5jjttgmfdsonnbv73cshadkd3c3m3dptcrhqjeds6xij6rlim.ipfs.dweb.link/))
+  - [V2](https://bafybeia5cxs72meianwphz2aq2tv5irpdgmkfniqsuhznl66s677zakgce.ipfs.dweb.link/))
+  - [V3](https://bafybeifgsqhf44fqxz4fnhomgwxy3666vrat2zu4djs5rgvjopatcfe42y.ipfs.dweb.link/)
```

Comment:
I feel like these should be `ipfs://` links, with perhaps a backup option using a centralized gateway.